android/assets/jsons/Civ V - Gods & Kings/Buildings.json

```diff
@@ -860,8 +860,8 @@
 	},
 	{
 		"name": "The Louvre",
-		"culture": 1,
-		"happiness": 4,
+		"culture": 4,
+		"greatPersonPoints": {"Great Artist": 2},
```

Comment:
Confirmed this is correct across Vanilla and Gods & Kings. Updated the Fandom:
https://civilization.fandom.com/wiki/Louvre_(Civ5)